src/tools/synth_node.rs

```diff
@@ -85,6 +86,23 @@ impl SyntheticNode {
         }
     }
 
+    /// Reads a message from the inbound (internal) queue of the node. If there is no message
+    /// by the given time there is an error returned indicating if timeout occured.
+    pub async fn recv_message_timeout(
+        &mut self,
+        duration: Duration,
+    ) -> Result<BinaryMessage, bool> {
+        let mut mess = None;
```

Comment:
`mess` sounds like a mess 😃
```suggestion
        let mut msg = None;
```